src/stories/components/ImageSelectionTask/ImageSelectionTask.js

```diff
@@ -17,21 +18,30 @@ import { Fragment } from 'react';
 export const ImageSelectionTask = ({ url }) => {
   return (
     <Fragment>
-      <p>In the following computer task, you will complete a series of
-        trials where you will be asked to make a split-second
-        choice between two images of yourself.</p>
-      <p>Each image is based on an actual photograph of you, but there is noise/
-        static on top of the photo that may alter its appearance.
-        The differences between the two images may be quite subtle,
-        so we ask that you go with your gut.</p>
-      <p>Two pictures will appear side by side on the screen.
-        For each pair of pictures, we would like you to select the picture that
-        is most like you. To select the image on the left side press the E key;
-        to select the image on the right press the I key.</p>
-      <p>This task is timed and will take approximately 20 minutes to complete.
-        There are no wrong answers, so GO AS FAST AS YOU CAN.
-        Responses that are too slow will not be recorded.</p>
-      <p>To get started, <a href={url}>click here</a>.</p>
+      {/* <p>In the following computer task, you will complete a series of*/}
+      {/*  trials where you will be asked to make a split-second*/}
+      {/*  choice between two images of yourself.</p>*/}
+      {/* <p>Each image is based on an actual photograph of you, but there */}
+      {/*  is noise/*/}
+      {/*  static on top of the photo that may alter its appearance.*/}
+      {/*  The differences between the two images may be quite subtle,*/}
+      {/*  so we ask that you go with your gut.</p>*/}
+      {/* <p>Two pictures will appear side by side on the screen.*/}
+      {/*  For each pair of pictures, we would like you to select the */}
+      {/*  picture that*/}
+      {/*  is most like you. To select the image on the left side press the */}
+      {/* E key;*/}
+      {/*  to select the image on the right press the I key.</p>*/}
+      {/* <p>This task is timed and will take approximately 20 minutes to */}
+      {/* complete.*/}
+      {/*  There are no wrong answers, so GO AS FAST AS YOU CAN.*/}
+      {/*  Responses that are too slow will not be recorded.</p>*/}
+      {/* <p>To get started, <a href={url}>click here</a>.</p>*/}
+
+      {/* TODO: Integrate JsPsych component into ImageSelectionTask to load on
```

Comment:
Temporarily including JsPysch here inside ImageSelectionTask so that people can test the experiment. Will have to find a home for JsPsych experiment in future. 

Is the idea that we will be redirecting the user to a different url to take experiment OR are we utilizing ScreenTakeOver for the experiment?

Reply:
Let's use the ScreenTakeOver component that Alessia wrote. Let's try to have it be conditionally rendered using a ternary operator or something like that. Let's add a button in ImageSelectionTask component that opens the ScreenTakeover with all of the jsPsych stuff once it's clicked.

Reply:
Sounds good to me! Let's create a separate ticket to implement that feature in the next sprint? 

I tried playing around with the ScreenTakeOver previously to render the JsPysch component, but I believe with the constraints of Experiment.js, it only renders on the right hand side rather than the full screen.

Reply:
I am ok with this 👍

Reply:
Note: Carlo has taken the responsibility for rendering JsPsych via ImageSelectionTask.js. There will be a separate task / PR for this feature.